src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/GetAllPaymentsForLastYearByAccountId.sql

```diff
@@ -0,0 +1,8 @@
+﻿CREATE PROCEDURE [employer_financial].[GetAllPaymentsForLastYearByAccountId]
+	@AccountId BIGINT
+AS
+	SELECT SUM(Amount)*-1 AS TotalSpend 
+	FROM employer_financial.TransactionLine 
+	WHERE TransactionDate >= DATEADD(YEAR,-1,GETDATE())
+	AND AccountId = @AccountId
```

Comment:
I think there should be another AND clause here to get Transactions which have a TransactionType of payment?

Reply:
There should indeed, don't know where that went as I remember writing it